### ISSUE TITLE
Codepipeline ARN not currently available

### ISSUE THREAD

**Issue by benfortuna** [terraform-providers/terraform-provider-aws#2714]
When creating a `code_commit_trigger` a `destination_arn` is required:

https://www.terraform.io/docs/providers/aws/r/code_commit_trigger.html#destination_arn

In the case of a `codepipeline` the destination is the ARN of the pipeline, however Terraform AWS Provider doesn't appear to provide this.

NOTE the arn is available in metadata via aws cli:

```
i$ aws codepipeline get-pipeline --name my_pipeline
{
    "metadata": {
        "created": 1513660917.19,
        "pipelineArn": "arn:aws:codepipeline:xxx-region-2:786326914961:my_pipeline",
        "updated": 1513726414.745
    },
   ...
```

Many thanks.

**Comment by bflad** [comment]
Hi @benfortuna thanks for reporting this! You are correct that we should be able to get the ARN from the AWS API/SDK and have Terraform know about it. I will implement a pull request for this shortly.

**Comment by bflad** [comment]
PR submitted: #2773

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.7.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#170-january-12-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
